src/systems/audio-zones-system.js

```diff
@@ -81,8 +96,20 @@ const updateSource = (function() {
       source.apply(inOutParams);
     } else {
       const params = outInParams;
-      params.gain = Math.min(outInParams.gain, inOutParams.gain);
-      params.coneOuterAngle = Math.min(outInParams.coneOuterAngle, inOutParams.coneOuterAngle);
+      if (outInParams.gain !== undefined && inOutParams.gain !== undefined) {
```

Comment:
Why are `gain` and `coneOuterAngle` the only properties that can change here? (I know this is not new behavior, but I don't know why it's this way.)

Reply:
This branch is covering the case where an audio is affected by an `inOut` Zone and also by an `outIn` Zone, so this case:

![audio-zones](https://user-images.githubusercontent.com/837184/135468038-b1491216-5313-45de-9ca9-fbcbe40c4095.jpg)

In this case the only clear thing to me at the time was that we want to use the `outIn` zone parameters as is the closest to the listener but still use the min gain between both. Not sure why also the conOuterAngle tbh, it doesn't make much sense now. I think we can probably remove that one.